helm/flink-operator/templates/flink-operator.yaml

```diff
@@ -191,3 +183,20 @@ data:
   {{- index (.Values.flinkDefaultConfiguration) "log4j-console.properties" | nindent 4 -}}
 {{- end }}
 {{- end }}
+---
+{{- if .Values.pvc.create }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.pvc.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "flink-operator.labels" . | nindent 4 }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Filesystem
+  resources:
+    requests:
+      storage: {{ .Values.pvc.size }}
+{{- end }}
```

Comment:
This is meant to create the `flink-userlib` pvc. If user specify the pvc mounts, then the flink-operator should be installed by `--set pvc.create=true`.  There seems no good way to align the pvc definition and the mounts. 
Since the mounts is a complex structure and may not be convenient to specify in the command line(From my use of helm, may be not right :), If user have to change the volume mount name , the pvc name should be changed correspondingly.

Reply:
Do you mean we do not have to in charge of the PVC creating when install operator?

Reply:
I think we could provide an example in the docs but maybe it would be easier to not bother with PV/PVC stuff, super complex and lot of options. Now with the mount configs users can mount any pvc easily.

Let's provide a simple example in the docs that works on minikube instead :)

Reply:
and we can remove the PVC section here completely

Reply:
I add an option to the `operatorVolumes` and `operatorVolumeMounts`.  I think the default mounts to hostPath should disable by default.